### PR TITLE
Add a script to print the local dependencies of a given podspec file.

### PR DESCRIPTION
**Pull request by wilhuff**
This implements my idea for pulling dependency information from the podspecs themselves.

### CODE REVIEW

**Comment by wilhuff** [comment]
Currently this only prints the list of dependencies on stdout. There are some differences with the list you've accumulated:

```bash
$ ruby scripts/pod_local_deps.rb FirebaseInAppMessagingDisplay.podspec
FirebaseAnalyticsInterop.podspec
FirebaseCore.podspec
FirebaseInAppMessaging.podspec
FirebaseInstanceID.podspec
GoogleUtilities.podspec
```
I believe your current implementation only finds GoogleUtilities, FirebaseCore, and FirebaseInAppMessaging. Was this intended?

**Comment by paulb777** [comment]
Thanks!  Your list looks complete and better than the partial list in my PR without an ongoing maintenance burden.

File: scripts/pod_local_deps.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env ruby
+
+# Copyright 2018 Google
```
Comment (paulb777) [inline scripts/pod_local_deps.rb]: 2019

Comment (wilhuff) [reply scripts/pod_local_deps.rb]: Done.

**Comment by wilhuff** [comment]
I wonder if this script is even necessary. Couldn't you pass `--include-podspecs="*.podspec"` to `pod lib lint`?

**Comment by paulb777** [comment]
It doesn't work to pass all the podspec's since the mac/tvOS specific ones will fail if they're given a podspec dependency that doesn't match architectures - although maybe that should be fixed in CocoaPods ....

**Pull request merged**